Dfe.Academies.External.Web/appsettings.Development.json

```diff
@@ -1,16 +1,20 @@
 {
-  "local_development": false,
-  "DetailedErrors": true,
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft.AspNetCore": "Warning"
-    }
-  },
-  "academies_api": {
-    "endpoint": "https://webapp-t1dv-sip-a2c.azurewebsites.net/"
-  },
-  "academisation_api": {
-    "endpoint": "https://webapp-t1dv-sip-a2c.azurewebsites.net/"
-  }
+	"local_development": false,
+	"DetailedErrors": true,
+	"Logging": {
+		"LogLevel": {
+			"Default": "Information",
+			"Microsoft.AspNetCore": "Warning"
+		}
+	},
+	"academies_api": {
+		"endpoint": "https://webapp-t1dv-sip-a2c.azurewebsites.net/"
```

Comment:
Seeing as though this and the `academisation_api:endpoint` values are duds, can we replace these URLs with just `"replace-me"` instead?

Reply:
Done - Don't worry about code changes